src/hooks/useConnectCurrentEvent.ts

```diff
@@ -9,7 +9,7 @@ export const useConnectCurrentEvent = () => {
   const { user } = useUser();
   const [currentTimestamp] = useState(Date.now() / 1000);
 
-  useFirestoreConnect([
+  useSparkleFirestoreConnect([
     {
       collection: "venues",
       doc: venueId,
```

Comment:
I am pretty sure this could potentially be `undefined` based on reading it from `useParams`, which would cause a pathologic query that loads all venues.